dto/event.dto.ts

```diff
@@ -0,0 +1,8 @@
+class EventDto {
+	queryStringParameters: any;
+	pathParameters: PathParameters | undefined;
+}
+
+class PathParameters {
+	imageName: string | undefined;
+}
```

Comment:
lib `@types/aws-lambda` provide all nessesary event types.

Reply:
In case of this repo you mostly use Funcional Programming style so that shouldn’t use class to define type.
should just use type
e.g
type EventDto = {
  queryStringParameters: Record<string, string>;
  pathParameters?: PathParameters;
}

TypeScript team recommend define type as above instead of using “interface”

should only use interface and class if you code in OOP style